modules/i18n/src/translations/en-US/portals/console.ts

```diff
@@ -616,8 +616,8 @@ export const console: ConsoleNS = {
                     deleteApplication: {
                         actionTitle: "Delete",
                         header: "Delete application",
-                        subheader: "The application will be deleted permanently and the " +
-                            "clients using this application will stop working."
+                        subheader: "Once the application is deleted, it cannot be recovered and the clients " +
+                            "using this application will no longer work. Please be certain."
```

Comment:
Do we need 'Please be certain.'?
Feels like it doesn't go with the rest of the wording flow.

Reply:
Removed 'Please be certain' phrase.